### PR TITLE
Move ctlplane tests to ctlplane dir

### PR DESCRIPTION
**Pull request by fao89**
It will reduce the conflicts on https://github.com/openstack-k8s-operators/openstack-operator/pull/765 when PRs get merged

### CODE REVIEW

**Comment by fao89** [comment]
@dprince I believe this is the core of the conflict between https://github.com/openstack-k8s-operators/openstack-operator/pull/792 and https://github.com/openstack-k8s-operators/openstack-operator/pull/765

I extracted this into this PR, because I had to rebase https://github.com/openstack-k8s-operators/openstack-operator/pull/765 every time a PR with tests is merged

**Comment by fao89** [comment]
recheck

File: apis/client/v1beta1/webhook_suite_test.go [hunk 1]
```diff
@@ -55,7 +55,7 @@ var (
 func TestAPIs(t *testing.T) {
 	RegisterFailHandler(Fail)
 
-	RunSpecs(t, "Webhook Suite")
+	RunSpecs(t, "OpenStackClient Webhook Suite")
```
Comment (dprince) [inline apis/client/v1beta1/webhook_suite_test.go:58]: This should still stay just 'Webhook suite' right?

**Comment by dprince** [comment]
one inline nit, otherwise looks good

**Comment by stuggi** [comment]
this has not run the kuttl tests - https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openstack-k8s-operators_openstack-operator/844/pull-ci-openstack-k8s-operators-openstack-operator-main-openstack-operator-build-deploy-kuttl/1801031141696737280/artifacts/openstack-operator-build-deploy-kuttl/openstack-k8s-operators-kuttl/build-log.txt

```
=== CONT  kuttl/harness/ctlplane
    logger.go:42: 23:55:31 | ctlplane | Ignoring ctlplane-basic-deployment as it does not match file name regexp: ^(\d+)-(?:[^\.]+)(?:\.yaml)?$
    logger.go:42: 23:55:31 | ctlplane | Ignoring ctlplane-collapsed as it does not match file name regexp: ^(\d+)-(?:[^\.]+)(?:\.yaml)?$
    logger.go:42: 23:55:31 | ctlplane | Ignoring ctlplane-galera-3replicas as it does not match file name regexp: ^(\d+)-(?:[^\.]+)(?:\.yaml)?$
    logger.go:42: 23:55:31 | ctlplane | Ignoring ctlplane-galera-basic as it does not match file name regexp: ^(\d+)-(?:[^\.]+)(?:\.yaml)?$
    logger.go:42: 23:55:31 | ctlplane | Skipping creation of user-supplied namespace: openstack-kuttl-tests
    logger.go:42: 23:55:31 | ctlplane | skipping kubernetes event logging
```

**Comment by fao89** [comment]
closing as https://github.com/openstack-k8s-operators/openstack-operator/pull/765 is passing now

**Pull request closed without merging**